app/models/test_track/config_updater.rb

```diff
@@ -1,5 +1,5 @@
 class TestTrack::ConfigUpdater
-  def initialize(schema_file_path = Rails.root.join('db', 'test_track_schema.yml'))
+  def initialize(schema_file_path = Rails.root.join('db/test_track_schema.yml'))
```

Comment:
I feel like we use the reverse of this config elsewhere -- I intentionally avoided pulling in `rubocop-betterment` to this repo to avoid blowing up too many things at once, but maybe it's time?